From 2e69c25ff31fe3ea38f140198d8582d7b2b7ec4a Mon Sep 17 00:00:00 2001
From: Didier Raboud <odyx@debian.org>
Date: Mon, 20 Jul 2015 08:23:50 +0200
Subject: [PATCH] Add maintscript stanzas to properly remove the mime.convs and
 mime.types conffiles in /etc/cups that are not shipped in this path since
 1.4.0~svn8773-1~

Closes: #792731
LP: #1475683
STR: #4593
---
 debian/cups.maintscript | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/debian/cups.maintscript b/debian/cups.maintscript
index 1640bc0..67a6e70 100644
--- a/debian/cups.maintscript
+++ b/debian/cups.maintscript
@@ -1,3 +1,5 @@
+rm_conffile /etc/cups/mime.convs 1.4.0~svn8773-1~
+rm_conffile /etc/cups/mime.types 1.4.0~svn8773-1~
 rm_conffile /etc/cups/acroread.conf 1.6.2-9~
 rm_conffile /etc/cups/pdftops.conf  1.6.2-9~
 rm_conffile /etc/modprobe.d/blacklist-cups-usblp.conf 1.5.3-2.1~
-- 
2.1.4

